database/migrations/2024_04_23_103334_add_user_info_to_orders.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('orders', function (Blueprint $table) {
+            $table->unsignedBigInteger('user_id');
```

Comment:
Om ervoor te zorgen dat de de relaties in de db goed worden gelegd:
```suggestion
            $table->foreignId('user_id')->constrained();
```